PhuongProject/ArrayCalculation.cs

```diff
@@ -20,10 +20,109 @@ public static int[] ArrayCalculate(int[] array1, int[] array2)
             int l2 = array2.Length;
             for (int i = 0; i < l2; i++)
             {
-               array1[array2[i]] += i;
+                array1[array2[i]] += i;
 
             }
             return array1;
         }
+
+        /// <summary>
+        /// Return the first 10 numbers of sum of all elements in an array
+        /// </summary>
+        /// <param name="array">The array</param>
+        /// <returns>The first 10 numbers of sum of all elements in an array</returns>
+        public static string FirstTenArraySum(string[] array)
```

Comment:
Add more methods to `ArrayCalculation` was a good choice.

Reply:
I don't understand your comment?  Do you compliment or criticize?

Reply:
Khen!